.github/workflows/sycl-windows-run-tests.yml

```diff
@@ -155,9 +183,32 @@ jobs:
         if [[ ${{inputs.compiler}} == 'icx' ]]; then
            export LIT_FILTER_OUT="compile_on_win_with_mdd"
         fi
-        export LIT_OPTS="-v --no-progress-bar --show-unsupported --show-pass --show-xfail --max-time 3600 --time-tests ${{ inputs.extra_lit_opts }}"
+        export LIT_OPTS="-v --no-progress-bar --show-unsupported --show-pass --show-xfail --max-time 3600 --time-tests --param test-mode=${{ inputs.e2e_testing_mode }} ${{ inputs.extra_lit_opts }}"
         cmake --build build-e2e --target check-sycl-e2e
 
+    # There are issues with running the tests with the same CMakeCache.txt, or
+    # even rebuilding it on top of an existing one in gen12, so we remove the
+    # configuration files before packing
```

Comment:
sorry can you explain what is going on here a little more?

Reply:
is it saying the ninja executable doesn't exist or is the error coming from somewhere inside the generated cmake files?

Reply:
ok yeah its a path issue, this solution seems better than fixing it